node_modules/oae-preview-processor/lib/processors/link/default.js

```diff
@@ -96,13 +98,16 @@ var generatePreviews = module.exports.generatePreviews = function(ctx, contentOb
              * All 3 of these would block embedding so as soon as the header is defined,
              * we add some metadata that tells the UI this link cannot be embedded.
              */
-            if (response.headers['x-frame-options']) {
+            if (response.headers['x-frame-options'] || forcedDownload) {
                 ctx.addPreviewMetadata('embeddable', false);
             } else {
                 ctx.addPreviewMetadata('embeddable', true);
             }
         }
 
+        var contentType = response && response.headers? response.headers['content-type']: null;
```

Comment:
I think it would be better to put this inside of the `else` block, so we don't execute this when there has been an error